# sensor parameters
# all refresh rates are given in hertz

# GPS parameters
gps:
  baud_rate: 115200
  enable: false
  serial_port: /dev/ttyUSB0
  refresh_rate: 10.0

# IMU parameters
imu:
  calibration_file_path: /home/corey/avc_ws/src/avc_sensors/config
  calibration_file_name: RTIMULib
  frame_id: imu_link
  refresh_rate: 10.0
  slerp_power: 0.02

# proximity sensor parameters
proximity_sensor:
  echo_pin: 4 # (using wiringPi numbering scheme)
  field_of_view: 0.3665 # (in radians)
  min_range: 0.5 # (in meters)
  max_range: 2.0 # (in meters)
  name: hc-sr04
  radiation_type: 0 # (0 = ultrasonic, 1 = infrared)
  refresh_rate: 10.0
  timeout: 25.0 # (in milliseconds)
  trigger_pin: 5 # (using wiringPi numbering scheme)
